projects/plugins/jetpack/modules/widgets/follow-button.php

```diff
@@ -64,23 +82,34 @@ class="wordpress-follow-button"
 			data-lang="<?php echo esc_attr( $wpcom_locale ); ?>"
 									<?php
 									if ( ! empty( $attributes ) ) {
-										echo implode( ' ', $attributes );}
-?>
+										echo implode( ' ', $attributes );} // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+									?>
 		>
-			<?php sprintf( __( 'Follow %s on WordPress.com', 'jetpack' ), get_bloginfo( 'name' ) ); ?>
+			<?php
+			/* Translators: %s is the site name. */
+			sprintf( __( 'Follow %s on WordPress.com', 'jetpack' ), get_bloginfo( 'name' ) );
```

Comment:
Looks like because there is no echo for the `sprintf`. The JS replaces the content with the follow button, but this widget isn't even shipped right now anyway.